### PR TITLE
Makes armor disks way rarer

### PR DESCRIPTION
**Pull request by genessee596**
## About The Pull Request

image

( everything is way too common )

## Why It's Good For The Game

These disks are supposed to be uncommon or rare, not omnipresent

## Changelog
:cl:
tweak: Armor design disks no longer dominate disk spawns.
/:cl:

### CODE REVIEW

**Comment by Iglpigl** [comment]
This still looks way too common to me, most rare items have rarity_value 70-100, but you should see what other people think too.

**Review by Evankhell561 — DISMISSED** [review]
I've been thinking about it and I would like to be left in charge of this. I am working on a logic update. which I will post soon.

**Comment by Evankhell561** [comment]
the spawn values ​​of the disks should be set in proportional values ​​to other disks. not to all items.

**Comment by genessee596** [comment]
> I finished my work and apparently in previous prs you made a mess with the spawn values ​​of the advanced disks and my new pr is not going to fix that.

The disks I "messed with" were broken and didn't spawn at all.

> the spawn values ​​of the disks should be set in proportional values ​​to other disks. not to all items.

Initially, the disks rarity was made equal to other disks' rarity. For whatever reason, they ended up cropping up way more than the disks with the same rarity.

**Comment by Evankhell561** [comment]
> > I finished my work and apparently in previous prs you made a mess with the spawn values ​​of the advanced disks and my new pr is not going to fix that.
> 
> The disks I "messed with" were broken and didn't spawn at all.
> 
> > the spawn values ​​of the disks should be set in proportional values ​​to other disks. not to all items.
> 
> Initially, the disks rarity was made equal to other disks' rarity. For whatever reason, they ended up cropping up way more than the disks with the same rarity.


1. Sorry if I sounded rude, I can't get used to the language.
2. That an object does not have a visible variable does not mean that it does not.
3. In your previous pr you lowered a rarity disc from 22 to 2 (which is wrong) but on the other hand you fixed the spawn value of the battery designs, thank you very much.

File: code/game/objects/items/weapons/autolathe_disks.dm [hunk 1]
```diff
@@ -1235,7 +1235,7 @@
 	disk_name = "Ironhammer Combat Equipment - Standard Armor"
 	icon_state = "ironhammer"
 	spawn_tags = SPAWN_TAG_DESING_ADVANCED_COMMON
-	rarity_value = 2 // one of the more common advanced disks
+	rarity_value = 5 // one of the more common advanced disks
```
Comment (Evankhell561) [inline code/game/objects/items/weapons/autolathe_disks.dm:1238]: 12

File: code/game/objects/items/weapons/autolathe_disks.dm [hunk 2]
```diff
@@ -1257,7 +1257,7 @@
 	disk_name = "Ironhammer Combat Equipment - Laserproof Armor"
 	icon_state = "ironhammer"
 	spawn_tags = SPAWN_TAG_DESING_ADVANCED_COMMON
-	rarity_value = 5.5 // slightly rarer than bulletproof gear
+	rarity_value = 12 // slightly rarer than bulletproof gear
```
Comment (Evankhell561) [inline code/game/objects/items/weapons/autolathe_disks.dm:1260]: 16

File: code/game/objects/items/weapons/autolathe_disks.dm [hunk 3]
```diff
@@ -1246,7 +1246,7 @@
 	disk_name = "Ironhammer Combat Equipment - Bulletproof Armor"
 	icon_state = "ironhammer"
 	spawn_tags = SPAWN_TAG_DESING_ADVANCED_COMMON
-	rarity_value = 5 // about as rare as a advanced tool disk - remember that this takes from the 'advanced' pool (which is rare) instead of the 'common' pool like the normal armor disk does
+	rarity_value = 10 // about as rare as a advanced tool disk - remember that this takes from the 'advanced' pool (which is rare) instead of the 'common' pool like the normal armor disk does
```
Comment (Evankhell561) [inline code/game/objects/items/weapons/autolathe_disks.dm:1249]: 15

**Review by Evankhell561 — CHANGES_REQUESTED** [review]

**Comment by genessee596** [comment]
Fixed to your specifications.

**Comment by Evankhell561** [comment]
> Fixed to your specifications.

perfect, if you don't mind working a bit more, can you raise the rarity of "/ obj / item / weapon / computer_hardware / hard_drive / portable / design / robustcellsdisk" to 2?

**Comment by Evankhell561** [comment]
5

> > Fixed to your specifications.
> 
> perfect, if you don't mind working a bit more, can you raise the rarity of "/ obj / item / weapon / computer_hardware / hard_drive / portable / design / robustcellsdisk" to 2.5?

edit: 2.5

**Comment by genessee596** [comment]
Ah cripes. Hold on.

**Comment by Evankhell561** [comment]
> Ah cripes. Hold on.

sorry.

**Comment by genessee596** [comment]
Fixed. 2.5 for the cells disk

**Review by Evankhell561 — APPROVED** [review]
ready

**Comment by genessee596** [comment]
Fix with maintainer approval. Merging now.

**Pull request merged**